recipes/azure-mgmt-synapse/recipe.yaml

```
@@ -0,0 +1,47 @@
+schema_version: 1
+
+context:
+  name: azure-mgmt-synapse
+  version: 2.0.0
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  - url: https://pypi.org/packages/source/${{ name[0] }}/${{ name }}/azure-mgmt-synapse-${{ version }}.zip
+    sha256: bec6bdfaeb55b4fdd159f2055e8875bf50a720bb0fce80a816e92a2359b898c8
+  - url: https://raw.githubusercontent.com/Azure/azure-sdk-for-python/70f714969c6311d02f41632785e2dc6d3595a826/LICENSE
```

That should be the correct license file.

```suggestion
  - url: https://raw.githubusercontent.com/Azure/azure-sdk-for-python/50471e7f07b430ab0bbc1984cd2fc102d908de41/sdk/synapse/azure-mgmt-synapse/LICENSE
```